flink-runtime/src/main/java/org/apache/flink/runtime/rest/RestClient.java

```diff
@@ -138,7 +138,7 @@ public RestClient(Configuration configuration, Executor executor)
                 final OutboundChannelHandlerFactory factory = factories.next();
                 if (factory != null) {
                     outboundChannelHandlerFactories.add(factory);
-                    LOG.info("Loaded channel outbound factory: {}", factory);
+                    LOG.debug("Loaded channel outbound factory: {}", factory);
```

Comment:
Please add some description to the commit message to why this is a good change. I'm completely lost here.

Reply:
This INFO log is too much, and not quite meaningful. I assume this is set by mistake for debugging: 
```
180248 [main] INFO  org.apache.flink.runtime.rest.RestClient [] - Loaded channel outbound factory: org.apache.flink.runtime.rest.Prio0OutboundChannelHandlerFactory@4b6e1c0
180248 [main] INFO  org.apache.flink.runtime.rest.RestClient [] - Loaded channel outbound factory: org.apache.flink.runtime.rest.Prio1OutboundChannelHandlerFactory@561b61ed
180254 [main] INFO  org.apache.flink.runtime.rest.RestClient [] - Loaded channel outbound factory: org.apache.flink.runtime.rest.Prio0OutboundChannelHandlerFactory@15400fff
180254 [main] INFO  org.apache.flink.runtime.rest.RestClient [] - Loaded channel outbound factory: org.apache.flink.runtime.rest.Prio1OutboundChannelHandlerFactory@18d910b3
180299 [main] INFO  org.apache.flink.runtime.rest.RestClient [] - Loaded channel outbound factory: org.apache.flink.runtime.rest.Prio0OutboundChannelHandlerFactory@77774571
180299 [main] INFO  org.apache.flink.runtime.rest.RestClient [] - Loaded channel outbound factory: org.apache.flink.runtime.rest.Prio1OutboundChannelHandlerFactory@277b8fa4
```

Reply:
We should leave INFO in as it's relevant for production use cases (e.g. authorization handler in platforms). 
That we have so much spam during tests is because we use test-jars of other projects where some test handlers are written. But we shouldn't make production harder to use because of our test setup.
@zentol may address that in a follow-up work - maybe we can leverage JUnit5 for that.

Reply:
Thanks for pointing this out. I removed this commit in the latest push.